src/aws/cloudwatch/log-group.ts

```diff
@@ -0,0 +1,834 @@
+// https://github.com/aws/aws-cdk/blob/v2.170.0/packages/aws-cdk-lib/aws-logs/lib/log-group.ts
+
+import { cloudwatchLogGroup } from "@cdktf/provider-aws";
+import { Annotations, Token } from "cdktf";
+import { Construct } from "constructs";
+import { ArnFormat, Arn, AwsSpec, AwsBeaconBase } from "..";
+import { DataProtectionPolicy } from "./data-protection-policy";
+import { LogStream } from "./log-stream";
+// import { MetricFilter } from "./metric-filter";
+import { FilterPattern, IFilterPattern } from "./pattern";
+import { ResourcePolicy } from "./policy";
+import {
+  ILogSubscriptionDestination,
+  SubscriptionFilter,
+} from "./subscription-filter";
+import * as iam from "../iam";
+// import * as kms from "../../encryption";
+
+export interface ILogGroup extends iam.IResourceWithPolicy {
```

Comment:
most of the PR errors are from my `WIP` (Work In Progress) commit I did originally (it was unfinished.. for example this should be `iam.IAwsBeaconWithPolicy`